exercises/practice/clock/.approaches/introduction.md

```diff
@@ -24,7 +24,7 @@ thus keeping the code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yoursel
 ```c
 #include <stdbool.h>
 
-#define MAX_STR_LEN 14 // "##:##\0" for int values
+#define MAX_STR_LEN (5 + 1) // "##:##\0" for int values
```

Comment:
```suggestion
#define MAX_STR_LEN sizeof("##:##")
```

Reply:
Today I learned

Reply:
I just learned this the other day. Very useful.